src/Controls/src/SourceGen/TypeConverters/RDSourceConverter.cs

```diff
@@ -55,7 +56,7 @@ static string GetResourcePath(string value, string rootTargetPath)
 
 		var uriType = context.Compilation.GetTypeByMetadataName("System.Uri")!;
 		var uriKindType = context.Compilation.GetTypeByMetadataName("System.UriKind")!;
-		context.Writer.WriteLine($"var {uriVar} = new {uriType.ToFQDisplayString()}(\"{value};assembly={asm.Name}\", {uriKindType.ToFQDisplayString()}.RelativeOrAbsolute);");
+		writer.WriteLine($"var {uriVar} = new {uriType.ToFQDisplayString()}(\"{value};assembly={asm.Name}\", {uriKindType.ToFQDisplayString()}.RelativeOrAbsolute);");
```

Comment:
This could be in a future PR, but you might save a few ns if you do this the ugly way, something like:
```csharp
writer.Write("var ");
writer.Write(uriVar);
writer.Write(" = new ");
writer.Write(uriType.ToFQDisplayString());
writer.Write("(\"");
writer.Write(value);
writer.Write(";assembly=");
writer.Write(uriKindType.ToFQDisplayString()}.RelativeOrAbsolute);
writer.WriteLine(';');
```
It would just avoid creating an intermediate string.

It might be worth doing these for the spots that are called a lot.